debian/control

```diff
@@ -0,0 +1,14 @@
+Source: sanoid
+Section: unknown
+Priority: optional
+Maintainer: Dustin L. Howett <dustin@howett.net>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.8
+Homepage: https://github.com/jimsalterjrs/sanoid
+Vcs-Git: https://github.com/jimsalterjrs/sanoid.git
+Vcs-Browser: https://github.com/jimsalterjrs/sanoid
+
+Package: sanoid
+Architecture: all
+Depends: ${misc:Depends}, ${perl:Depends}, zfsutils-linux, libconfig-inifiles-perl
```

Comment:
Other perl packages might have to go here; `perl:depends` expands to `perl`, really, so the package gets a perl dependency.

Reply:
can you replace "zfsutils-linux" with "zfsutils-linux | zfs", then it would also work with custom debian packages generated from the zfs git repository

Reply:
@phreaker0 , I made your change - please review and make sure syntax is appropriate if you have a system which would benefit from it; I'm a complete newb with debian packaging.

Reply:
thx, i verified that the generated sanoid package works for both the official zfs packages provided by debian/ubuntu and the custom debian packaged generated from the official zfs git repository